### ISSUE TITLE
Slidr resizes content instead of pushing it on mobile devices

### ISSUE THREAD

**Issue by kevyduan** [artberri/sidr#198]
Hi guys! There's an issue I've discovered that only affects mobile devices. On desktops and even when I resize my desktop sidr works fine. It pops out and pushes the content to the left. However on mobile devices the sidr menu comes out however instead of pushing the content to the left, it resizes it and maxes it smaller. I've seen this bug happen not only on my site but on sidr's own site: http://www.berriart.com/sidr/ Is there any fix to this issue? Thanks!

**Comment by KZeni** [comment]
Seems this viewport issue is also present in iOS 9 (beta).

Opening the Sidr menu on the left pushes the body over, but iOS 9 (beta) Safari also scales the site to fit the menu and (now slid-over) site even if you have the viewport set to device-width and disabled user scaling. Meanwhile, opening a Sidr menu on the right doesn't have this problem (likely due to how having elements overflow on the left and/or top doesn't extend the canvas).

Either way, I believe I've found a fix to make it keep the site's scale/viewport the same in both cases.

I've posted a pull request with the quick & simple fix here: #268
